entry.sh

```diff
@@ -72,9 +72,14 @@ function init_non_systemd()
 	# echo error message, when executable file doesn't exist.
 	if [ $? == '0' ]; then
 		shift
-		"$CMD" "$@" &
-		pid=$!
-		wait $pid
+		if [ ! -z "$RESIN_SUPERVISOR_API_KEY" ] && [ ! -z "$RESIN_DEVICE_UUID" ]; then
```

Comment:
It's probably best to check for `RESIN` - https://github.com/resin-io/resin-supervisor/blob/master/src/utils.coffee#L163